aviary/subsystems/propulsion/propeller/hamilton_standard.py

```diff
@@ -829,7 +841,7 @@ def setup(self):
         nn = self.options['num_nodes']
 
         self.add_input(Aircraft.Engine.PROPELLER_DIAMETER, val=0.0, units='ft')
-        self.add_input(Dynamic.Mission.INSTALLATION_LOSS_FACTOR,
+        self.add_input('install_loss_factor',
```

Comment:
Is there a reason we are moving away from Dynamic.Mission.INSTALLATION_LOSS_FACTOR?

Reply:
I didn't feel like it was important enough as a variable to track in the hierarchy - it's an intermediate calculation value that doesn't seem important enough to warrant adding to the hierarchy... I'm open to debate on what does/does not belong in the hierarchy

Reply:
Agreed it's not important enough to warrant metadata inclusion

Reply:
after looking for the other variables, I noticed that you can set this directly from the input deck or have it calculated by the code. I'm not sure if that influences your thoughts on inclusion